client/toError.js

```diff
@@ -1,7 +1,8 @@
-const toError = ({ message, name, status }) => {
+const toError = ({ message, name, status, data }) => {
```

Comment:
I'm a stickler for alphabets.  Let's keep these sorted alphabetically to make things easier to spot visually.